type-equality-compat/src-old/Data/Type/Equality.hs

```diff
@@ -28,6 +28,12 @@ module Data.Type.Equality (
     apply,
 #endif
 
+    -- inner and outer not implemented,
```

Comment:
Actually, you _can_ implement `inner` back to 7.0 (and `outer` back to 7.6) using the Leibniz equality trick. See https://gitlab.haskell.org/snippets/1464 for a proof of concept.